### PR TITLE
Fix user list items overflow

### PR DESCRIPTION
**Pull request by veymarjoel**

### CODE REVIEW

File: src/containers/Profile/profile.css [hunk 1]
```diff
@@ -1,3 +1,9 @@
 .profile .lateral-container {
   width: 100% !important;
+}
+
+.profile .main-cards {
```
Comment (albertodarblade) [inline src/containers/Profile/profile.css]: Its nesesary to use important property??

Comment (veymarjoel) [reply src/containers/Profile/profile.css]: the important part was removed thanks!

File: src/components/SmartCard/SmartCard.js [hunk 2]
```diff
@@ -69,7 +69,7 @@ class SmartCard extends React.Component {
       marginWithImage = 'margin-image';
     }
     return (
-      <Card.Content>
+      <Card.Content >
```
Comment (cinthia-neko) [inline src/components/SmartCard/SmartCard.js]: this space is necessary?

Comment (veymarjoel) [reply src/components/SmartCard/SmartCard.js]: done thanks!

Comment (veymarjoel) [reply src/components/SmartCard/SmartCard.js]: done thanks!

**Pull request merged**